graphql/types/Shipping.graphql

```diff
@@ -9,20 +9,31 @@ type Shipping {
 
 type DeliveryOption {
   id: String!
+  deliveryChannel: String!
   price: Int!
   estimate: String!
   isSelected: Boolean!
 }
 
 type PickupOption {
   id: String!
+  address: Address!
+  deliveryChannel: String!
   price: Int!
   estimate: String!
   isSelected: Boolean!
   friendlyName: String!
```

Comment:
Could you check which values the API can return here, so we can use an enum instead? (cc @lucasecdb wdyt?)

Reply:
which field do you mean?

Reply:
`friendlyName`

Reply:
it is an open field, we can't use an enum here (e.g. "Loja no Flamengo do Rio de Janeiro" is a friendly name)

Reply:
uh oh, I meant `deliveryChannel`

Reply:
Mostly because of this loc: https://github.com/vtex-apps/checkout-shipping/pull/18/files#diff-094884431539f7d7efcec90636cbd36c3bc112298a412123bae7cbb465da940bR6

> ```shippingOption.deliveryChannel === 'pickup-in-point'```

Reply:
But we are dealing with two different types (DeliveryOption and Pickup Option) with only one possible value each. I don't know if this would help us. Maybe we can do that later. Wdyt?

Reply:
I think this would be much more worthy if we had a `ShippingOption` type that could have both `delivery` and `pickup-in-point` as its `deliveryChannel`.